src/Components/Links.js

```diff
@@ -8,6 +8,7 @@ function Links({ links }) {
     youtube: '#FF0000',
     twitter: '#00ACEE',
     github: '#171515',
+    linkedin: '#0a66c2',
```

Comment:
Please Add ScreenShot of Your Version on Deployment So it will be easy to check how it looks

Reply:
![Screenshot (1)](https://user-images.githubusercontent.com/22175273/133917953-1afe85a8-cec2-4744-be8f-80137f60c211.png)
You can check this image and verify